src/test/java/io/vertx/core/impl/launcher/commands/RunCommandTest.java

```diff
@@ -183,6 +183,20 @@ public void testWithConfProvidedInline() throws IOException {
   }
 
   @Test
+  public void testWithBrokenConfProvidedInline() throws IOException {
+    setManifest("MANIFEST-Launcher-Http-Verticle.MF");
+    cli.dispatch(new Launcher(), new String[] {"--conf={\"name\":\"vertx\""});
```

Comment:
you could generate a random alpha string for config and check that this particular string is not in the output.